src/common/components/Heading/Heading.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export const Heading = (props) => {
+  const level = props.level || 2
```

Comment:
In React, we can use `defaultProps` instead! Whenever a prop is not provided, we get the default.

```
Heading.defaultProps = {
  level: 2
}
```